frontend/src/components/homepage-features.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import clsx from 'clsx'
+import styles from './HomepageFeatures.module.css'
+
+const FeatureList = [
+  {
+    title: 'Dynamic badges',
+    description: (
+      <>
+        <img
+          alt="build:passing"
+          src="https://img.shields.io/badge/build-passing-brightgreen"
+        />
+        <br />
+        Show metrics for your project. We've got badges for hundreds of
+        services.
+      </>
+    ),
+  },
+  {
+    title: 'Static Badges',
+    description: (
+      <>
+        Create a badge with
+        <br />
+        <img
+          alt="any text you like"
+          src="https://img.shields.io/badge/any%20text-you%20like-blue"
+        />
+      </>
+    ),
+  },
+  {
+    title: 'Badge-Maker NPM library',
+    description: (
+      <>
+        Render badges in your own application using our{' '}
+        <a href="https://www.npmjs.com/package/badge-maker">NPM library</a>
+        <br />
+        <code>npm install badge-maker</code>
+      </>
+    ),
+  },
+  {
+    title: 'Host your own instance',
+    description: (
+      <>
+        Host a shields instance behind your firewall with our{' '}
+        <a href="https://github.com/badges/shields/pkgs/container/shields">
+          docker image
+        </a>
+        <br />
+        <code>docker pull ghcr.io/badges/shields</code>
```

Comment:
Maybe roll back https://github.com/badges/shields/pull/9014/commits/ef74cbb26b1c24ce987a8975e60313682f9161f8

https://www.docker.com/blog/no-longer-sunsetting-the-free-team-plan/